prowler/providers/aws/lib/security_hub/security_hub.py

```diff
@@ -354,6 +354,52 @@ def batch_send_to_security_hub(
             )
         return success_count
 
+    def get_existing_findings_timestamps(self) -> dict:
+        """
+        Retrieves existing findings from Security Hub and returns their timestamps.
+
+        Returns:
+            dict: A dictionary mapping finding IDs to their timestamps from Security Hub.
+        """
+        existing_findings_timestamps = {}
+        logger.info("Retrieving existing findings timestamps from Security Hub.")
+
+        for region in self._findings_per_region.keys():
+            try:
+                # Get findings of that region
+                findings_filter = {
+                    "ProductName": [{"Value": "Prowler", "Comparison": "EQUALS"}],
+                    "RecordState": [{"Value": "ACTIVE", "Comparison": "EQUALS"}],
+                    "AwsAccountId": [
+                        {"Value": self._aws_account_id, "Comparison": "EQUALS"}
+                    ],
+                    "Region": [{"Value": region, "Comparison": "EQUALS"}],
```

Comment:
You can add another filter here to look for the scan findings:
```
"Id": [{"Value": finding1.uid, "Comparison": "EQUALS"}, {"Value": finding2.uid, "Comparison": "EQUALS"}],
```
With the above you don't need to call this function before.